modules/ROOT/pages/dashboard-config-ref.adoc

```diff
@@ -255,7 +381,7 @@ Not in UI anymore:
 
 [[samples_markdown]]
 == Markdown Syntax Support
-Some Anypoint Monitoring fields, such as Text graph fields in a custom dashboard, accept Markdown. Markdown is a lightweight syntax for styling all forms of writing.
+Some Anypoint Monitoring fields, such as text graph fields in a custom dashboard, accept Markdown. Markdown is a lightweight syntax for styling all forms of writing.
```

Comment:
Please remove this Markdown section. There's nothing at all to be gained by documenting something easily found on the web. Just add a link to https://www.markdownguide.org/basic-syntax/.

Reply:
Beyond the scope of this PR. But I will do it in the future.